internal/processor/otel/traces.go

```diff
@@ -649,6 +651,8 @@ func TranslateSpan(spanKind ptrace.SpanKind, attributes pcommon.Map, event *mode
 			case semconv.AttributeRPCService:
 				rpcService = stringval
 				foundSpanType = rpcSpan
+			case semconv.AttributeRPCGRPCStatusCode:
+				event.Transaction.Type = "request"
```

Comment:
I just noticed that this code is meant for translating spans, so we shouldn't be setting transaction type.

```suggestion
				rpcSystem = "grpc"
				foundSpanType = rpcSpan
```